features/crashprobe.feature

```diff
@@ -1,91 +1,81 @@
 Feature: Reporting crash events
 
-Background:
-    Given I set environment variable "BUGSNAG_API_KEY" to "a35a2a72bd230ac0aa0f52715bbdc6aa"
-
 Scenario: Executing privileged instruction
-    When I crash the app using "PrivilegedInstructionScenario"
-    And I relaunch the app
-    And I wait for a request
-    Then the request is valid for the error reporting API
-    And the "Bugsnag-API-Key" header equals "a35a2a72bd230ac0aa0f52715bbdc6aa"
-    And the payload field "notifier.name" equals "iOS Bugsnag Notifier"
-    And the payload field "events" is an array with 1 element
-    And the exception "errorClass" equals "EXC_BAD_ACCESS"
+    When I run "PrivilegedInstructionScenario" and relaunch the app
+    And I configure Bugsnag for "PrivilegedInstructionScenario"
+    And I wait to receive a request
+    Then the request is valid for the error reporting API version "4.0" for the "iOS Bugsnag Notifier" notifier
+    And the payload field "events" is an array with 1 elements
+    And the exception "errorClass" equals "EXC_BAD_INSTRUCTION"
```

Comment:
What's happening with the expected error classes in the first three scenarios here?  `EXC_BAD_ACCESS` changed to `EXC_BAD_INSTRUCTION` and so on.